packages/calcite-components/src/components/panel/panel.tsx

```diff
@@ -326,7 +341,8 @@ export class Panel
    *   top: 0, // Specifies the number of pixels along the Y axis to scroll the window or element
    *   behavior: "auto" // Specifies whether the scrolling should animate smoothly (smooth), or happen instantly in a single jump (auto, the default value).
    * });
-   * @param options
+   * @param options - allows specific coordinates to be defined.
+   * @returns - promise.
```

Comment:
I think this needs a bit more info. WDYT about `promise that resolves once the content is scrolled to`? + @geospatialem for editorial suggestions.

Applies to other `@returns` with the same description.

Reply:
Cool. changed and awaiting @geospatialem's approval

Reply:
![seal-approval](https://github.com/Esri/calcite-design-system/assets/5023024/98e2fd31-044c-4678-8701-3f7588ad3e8e)